x/auth/client/cli/tx_sign.go

```diff
@@ -243,15 +242,44 @@ func makeSignCmd() func(cmd *cobra.Command, args []string) error {
 		}
 
 		overwrite, _ := f.GetBool(flagOverwrite)
-		if multisig != "" {
-			multisigAddr, err := sdk.AccAddressFromBech32(multisig)
```

Comment:
It seems to me that this line got deleted. @likhita-809 is this intended?

Reply:
yes, because at the line L251, we are getting both name and address of the multisig irrespective of multisig flag value(be it name or address).